packages/appstore/src/components/containers/__tests__/listing-container.spec.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ListingContainer from '../listing-container';
+import { StoreProvider, mockStore } from '@deriv/stores';
+
+jest.mock('Components/pre-loader/title-card-loader', () => jest.fn(() => <span>TitleCardLoader</span>));
+jest.mock('Components/currency-switcher-card', () => jest.fn(() => <span>CurrencySwitcherCard</span>));
+jest.mock('Components/containers/grid-container', () => jest.fn(({ children }) => <>{children}</>));
+
+const Title = () => <span>Test Title</span>;
+const Description = () => <span>Test Description</span>;
+
+let mock = mockStore({});
```

Comment:
```suggestion
```

Reply:
@farzin-deriv I have a common wrapper function for all test cases, if I do so as you want I'll need to add wrapper function for every test cases too. In my opinion, it's not a good idea to duplicate the code. What do you think?

Reply:
@sergei-deriv It's a test file, duplicated code shouldn't be a concern to sacrifice the test quality. It's better to isolate each test case to itself and not dependent on global things, That's why we added `mockStore` to generate a new fresh mock object for each test case.

Anyway, This is totally personal preference and there is no best practice, Some devs prefer to create things once at the top of the test file and reuse them, And some others prefer to isolate each test case 😅

I've already approved the PR so the comments are just suggestions, You can ignore them 🙇🏻